### DISCUSSION TITLE
High precision mouse setting should be included in first-time setup

### DISCUSSION THREAD

**Discussion by Bubbleman532** [ppy/osu#25122] [Ideas]
While this setting is usually better toggled on for mouse, in stable it is toggled off by default.

![image](https://github.com/ppy/osu/assets/117375368/da214adc-31ba-4f9b-8f7e-8c03d690bae3)

At minimum the default should be off, but this setting could/should be added to the first-time setup screen with a description; especially since external tablet drivers will conflict with the in-built drivers and it took me a few minutes to figure out what exactly was going on with my aim when I was setting up my client at COE.

**Comment by peppy** [comment]
I have it on my short list to improve the initial input setup, which this will definitely be part of.

**Comment by Stoppedpuma** [comment]
A key toggle for this during setup would be nice as I've had an issue where my cursor would just lock to the top left making it difficult to disable.
